src/app-layout/__tests__/global-breadcrumbs.test.tsx

```diff
@@ -122,23 +133,54 @@ describeEachAppLayout({ themes: ['refresh-toolbar'], sizes: ['desktop'] }, () =>
     expect(findRootBreadcrumb().getElement()).toHaveTextContent('Second');
   });
 
-  test('when multiple app layouts rendered, only the first instance receives breadcrumbs', async () => {
+  test('when multiple app layouts rendered, only the last instance receives breadcrumbs', async () => {
```

Comment:
Expected change. New PR revisits the deduplication logic, the last instance renders the deduplicated stuff. This behavior is better aligned with the visible UI structure